src/vue/components/info.vue

```diff
@@ -52,7 +61,15 @@ export default {
   computed: {
     defaultLanguage () {
       return this.app.getLanguageName(this.app.getDefaultLangCode()).name
+    },
+    faqLink() {
+      if (this.$store.state.app.embedLibActive) {
+        return "https://alpheios.net/pages/v3/faq-embedded"
+      } else {
+        return "https://alpheios.net/pages/v3/faq-extension"
+      }
     }
+
```

Comment:
I have checked - these links return 404 Not Found
And I was testing it in embed-lib - it redirects to https://alpheios.net/pages/v3/faq-extension